pkg/machines/libvirt-dbus.es6

```diff
@@ -515,6 +519,21 @@ LIBVIRT_DBUS_PROVIDER = {
         };
     },
 
+    GET_NODE_MAX_MEMORY({ connectionName }) {
+        logDebug(`${this.name}.GET_NODE_MAX_MEMORY: connection: ${connectionName}`);
+
+        if (connectionName) {
+            return dispatch => call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'NodeGetMemoryStats', [0, 0], TIMEOUT)
+                    .then((stats) => {
+                        const memory = convertToUnit(stats[0].total, 'KiB', 'MiB');
+                        dispatch(setNodeMaxMemory({ memory, connectionName }));
+                    })
+                    .fail(ex => console.error("NodeGetMemoryStats failed: %s", ex));
+        }
+
+        return unknownConnectionName(setNodeMaxMemory);
```

Comment:
I don't think this is needed here. I see it exists in some other places as well, but please don't copy blindly code :)